ui_src/src/resolvers.js

```diff
@@ -10,21 +10,21 @@ function dnaToUiNote (noteResult) {
 export const resolvers = {
   Query: {
     getNote: async (_, { id }) =>
-      dnaToUiNote(await createZomeCall('/react-graphql/notes/get_note')({ id })),
+      dnaToUiNote(await createZomeCall('/reactGraphql/notes/get_note')({ id })),
```

Comment:
can we change the dna instance id to something more specific? even `notes` seems fine to me. so `notes/notes/get_note` etc.

Reply:
Yep, I'd like the instance to be the dna name, in this case notes.

Reply:
Done.